### DISCUSSION TITLE
ETA for general availability?

### DISCUSSION THREAD

**Discussion by snebel29** [awslabs/aws-sdk-rust#686] [General]
Hi,
We are considering using by the S3  portion of this SDK to build a production ready service and I wonder if there is an indication of its general availability release.

Alternatives, as a suggestion and until GA is released would be much appreciated.

Cheers

**Comment by jdisanti** [comment]
I don't have any timelines that I can share, but we are tracking production readiness in https://github.com/awslabs/smithy-rs/issues/1692.

**Reply by snebel29** [reply]
Thank is a great link, thank you!

**Comment by MartinMcCaulley** [comment]
Any way to give a general indication of when the GA release will be available? Within 3 - 4 months?

Thank you for all the great work on this SDK!

Best regards
